app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,412 @@
+@import 'bootstrap';
+* {
+  box-sizing: inherit;
+}
+html {
+  margin: 0;
+  padding: 0;
+}
+body {
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+}
+
+#navbar-nav ul li {
+  text-align: center;
+}
+.jumbotron {
+  background: none;
+  margin-bottom: 0;
+}
+.navbar {
+  padding-left: 10%;
+  padding-right: 10%;
+}
+.texty {
+  color: white !important;
+  text-shadow: 3px 3px 10px #ccc, 3px 3px 10px #000, -1px 0 black, 0 1px black, 1px 0 black, 0 -1px black;
+  font-family: Verdana !important;
+  font-weight: 500;
+}
+#logo-text {
+  padding: 4px 5px;
+  border: 0px;
+  line-height: 12px;
+  height: 4px;
+  font-size: 11px !important;
+  font-weight: 500;
+  font-family: Verdana !important;
+  letter-spacing: 1px;
+}
+.logo-img {
+  width: 30px;
+  height: 30px;
+  border-radius: 15px;
+  box-shadow: 0px 0px 10px black;
+}
+#menu-top li {
+  font-family: Verdana !important;
+  line-height: 8px;
+  font-size: 11px;
+  letter-spacing: 2px;
+  transition: transform .2s;
+}
+.title, .description {
+  font-family: Verdana !important;
```

Comment:
Just use this as default font, don't add it everywhere. And use a variable for this. Please read about bootstrap typography